cmd/podman/containers_prune.go

```diff
@@ -47,7 +56,7 @@ func pruneContainers(runtime *adapter.LocalRuntime, ctx context.Context, maxWork
 	for _, container := range delContainers {
 		con := container
 		f := func() error {
-			return runtime.RemoveContainer(ctx, con, force)
+			return runtime.RemoveContainer(ctx, con, false)
```

Comment:
Force -> unconditional false?

I think this is correct (I don't see a force flag), but just making sure

Reply:
fixed